plugins/c8y_configuration_plugin/src/main.rs

```diff
@@ -34,10 +42,10 @@ async fn main() -> Result<(), anyhow::Error> {
     let msg = plugin_config.to_message()?;
     let () = mqtt_client.published.send(msg).await?;
 
-    // mqtt loop
+    // Mqtt message loop
     while let Some(message) = mqtt_client.received.next().await {
         debug!("Received {:?}", message);
-        match message.payload_str()?.split_at(3).0 {
+        match message.payload_str()?.split(',').nth(0).unwrap_or_default() {
```

Comment:
Would be a good idea to log the `default` case in these match arms as "unsupported config operation".